appveyor.yml

```diff
@@ -14,6 +14,7 @@ cache:
 
 init:
   - ps: $Env:path = $Env:python + ";" + $Env:python + "\scripts;" + "C:\Program Files\Microsoft SDKs\Windows\v7.0\Bin;" + "C:\vtk\bin;" + $Env:path
+  - ps: $env:ETS_TOOLKIT="qt4"
```

Comment:
Actually, this line should not be needed. It looks that the new `test_scene` is doing things wrong.